frontend/src/metabase/timelines/components/TimelineCard/TimelineCard.tsx

```diff
@@ -18,15 +20,27 @@ const TimelineCard = ({
   timeline,
   collection,
 }: TimelineCardProps): JSX.Element => {
+  const eventsCount = timeline.events?.length;
+  const hasDescription = Boolean(timeline.description);
+
   return (
     <CardRoot to={Urls.timelineInCollection(timeline, collection)}>
       <CardIcon name={timeline.icon} />
       <CardBody>
         <CardTitle>{timeline.name}</CardTitle>
         <CardDescription>{timeline.description}</CardDescription>
       </CardBody>
+      {eventsCount != null && (
```

Comment:
Should we maybe use something like `Number.isSafeInteger` as we really need an int here?

Reply:
This is actually checking if `timeline.events` is not null. It generally shouldn't be, but in case of another request overriding the redux state it might become null.